src/test/java/org/apache/commons/collections4/IterableUtilsTest.java

```diff
@@ -274,7 +274,7 @@ public void find() {
         assertTrue(test == null);
         assertNull(IterableUtils.find(null,testPredicate));
         try {
-            assertNull(IterableUtils.find(iterableA, null));
```

Comment:
The `assertNull` here was really confusing. I would expect that it would assert confirming that something **is null**. But instead the `.find` would throw a NPE, that it caught below. i.e. no need to `assertNull` if I understand it right.